tests/Intl/Grapheme/GraphemeTest.php

```diff
@@ -89,6 +89,10 @@ public function testGraphemeSubstr()
     {
         $c = 'déjà';
 
+        if (\PHP_VERSION_ID >= 70000) {
```

Comment:
why only on PHP 7.0+ when the polyfill does not have such condition ?

Reply:
reverted: the behavior depends also on the version of the intl extensions. The implementation is now unconditional (but tests are for practical reasons)